test/cli/kit/resolver_test.rb

```diff
@@ -36,8 +36,8 @@ def test_resolver_no_match
       end
 
       def test_resolver_suggest
-        reg.add(32, 'floo')
```

Comment:
Could you please revert the changes in this file? The previous did a better job of showing ranked proximity to the provided command.

Reply:
The problem I was solving is that the current two examples have the same ranking. On Windows they get returned in different orders. I guess the order is undefined when they have the same ranking.